src/core/renderer/hostDescriptors/descriptors/geometries/geometry.ts

```diff
@@ -44,6 +49,16 @@ class GeometryDescriptor extends WrappedEntityDescriptor<GeometryWrapper,
       instance.vertices = newValue;
       instance.verticesNeedUpdate = true;
     }, false, true);
+
+    this.hasProp<Face3[]>("faces", (instance, newValue, oldProps, newProps) => {
+      if (instance.faces.length !== newValue.length) {
+        this.remountTrigger(instance, newValue, oldProps, newProps);
```

Comment:
I saw that `vertices` required a `remountTrigger` so I did it here as well. When does a prop require a `remountTrigger`?